api/rosetta/data_balance_integration_test.go

```diff
@@ -134,9 +71,8 @@ func TestGetBalance(t *testing.T) {
 
 		request rosetta.BalanceRequest
 
-		wantStatusCode int
-		wantBalance    string
-		validateBlock  blockIDValidationFn
+		wantBalance   string
+		validateBlock blockIDValidationFn
```

Comment:
My rationale was that we're verifying the block ID only (index+hash), and we also have the block as in the output of the `/block` endpoint, so wanted to make it clear what's being checked..

Reply:
I do know your rationale, but that's Java / C# rationale. The rationale in Go is, if it's not ambiguous, it's fine. Whether we validate only the ID of the block or the whole block or any other part of the block isn't really relevant, if that's the only part of the block we validate. And the _purpose_ of the function is more important than the _implementation_.

Reply:
Which is super weird since I've never done either 😄 

I'll try to keep that in mind. It's weird because it seems like the proper names you suggest are pretty much the names I'd use when writing C, but I keep trying to think of "what would be more Go-like" and end up with this weird, verbose variant.